src/bp/nlu-core/intents/svm-intent-classifier.ts

```diff
@@ -111,7 +111,7 @@ export class SvmIntentClassifier implements IntentClassifier {
 
       const confidence = 1 / intentNames.length
       return {
-        intents: intentNames.map(ctx => ({ name: ctx, confidence }))
+        intents: intentNames.map(ctx => ({ name: ctx, confidence, extractor: 'svm-classifier' }))
```

Comment:
that case is kindof weird.. how does this happen ? then there is not svm predictor and there is more than 0 intents ... basically 1 intent .. other than that it wouldn't predict correct ?

Reply:
Yeah you're right. There is only 1 intent here. 

I can rewrite using `intentNames[0]` here + rename the variable `ctx` which is a leftover.

```
intents: 
```